app/uk/gov/hmrc/helptosavefrontend/controllers/EligibilityCheckController.scala

```diff
@@ -184,7 +184,11 @@ class EligibilityCheckController @Inject() (
   def getMissingInfoPage: Action[AnyContent] =
     authorisedForHtsWithInfo { implicit request => implicit htsContext =>
       htsContext.userDetails.fold(
-        missingInfo => Ok(missingUserInfo(missingInfo.missingInfo)),
+        { missingInfo =>
+          // DLS-10416 - Check in Kibana to see whether we can remove the Date of Birth warning
+          logger.info("Missing user info: " + missingInfo.missingInfo.toString())
```

Comment:
Same as above, would be ideal not to log all details, but the fact that DOB is missing

Reply:
Discussion in the other message, nothing to address now